flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkV2Branch.java

```diff
@@ -46,12 +46,19 @@ public class TestFlinkIcebergSinkV2Branch extends TestFlinkIcebergSinkV2Base {
   private static final HadoopCatalogExtension CATALOG_EXTENSION =
       new HadoopCatalogExtension(DATABASE, TestFixtures.TABLE);
 
-  @Parameter(index = 0)
-  private String branch;
+  @Parameter(index = 4)
+  protected String branch;
 
-  @Parameters(name = "branch = {0}")
+  @Parameters(name = "branch = {4}")
```

Comment:
the parameter names from the parent class should probably be included here

Reply:
thanks so much for applying this!